src/extensions.rs

```diff
@@ -51,6 +52,22 @@ impl CommandExt for Command {
 
         String::from_utf8(out.stdout).wrap_err_with(|| format!("`{:?}` output was not UTF-8", self))
     }
+
+    /// Runs the command to completion and returns the status and its stdout.
+    fn run_and_get_status_and_stdout(&mut self, verbose: bool) -> Result<(ExitStatus, String)> {
```

Comment:
in #684 I've made something like this. https://github.com/cross-rs/cross/blob/53c883b3ff0ab310a9fdb48dd5c7cb9c1a098b54/src/extensions.rs#L56

I think instead of a `run_and_get_status_and_stdout` function, `run_and_get_output` + an extension on `Output` to add 

```rust
fn stdout(&self) -> Result<String> {
    String::from_utf8(out.stdout).wrap_err_with(|| format!("`{:?}` output was not UTF-8", self)) }
```

makes more sense

Reply:
We could probably then implement `run_and_get_stdout` in terms of this as well. Maybe you could file this in a separate PR and we can rebase this PR and #684 off of that?